descriptors/ModuleDescriptor-template.json

```diff
@@ -564,6 +581,7 @@
       "subPermissions": [
         "inventory-storage.authorities.item.get",
         "inventory-storage.authorities.item.post",
+        "inventory-storage.authorities.bulk.post",
```

Comment:
(not related to this story) should the permission ```inventory-storage.authorities.collection.get``` be part of this set of permissions?